.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
   "name": "Container",
   // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-  "image": "jelaniwoods/appdev2023-rails-template",
+  "image": "jelaniwoods/appdev2023-rails-template-database-url-test",
```

Comment:
I guess you'll drop the `-database-url-test` on final merge?

Reply:
Correct. When I make a change to a Docker image, I'll usually push to a _different_ image. That way, if anything breaks, my changes don't affect current students and I have time to fix it.

Since it's been tested, I'll tag the image with the usual name and push the changes so it affects current students.